code/upaya/app/assets/javascripts/generating_quiz.js.erb

```diff
@@ -0,0 +1,9 @@
+$(document).on('page:change', function () {
+  $('#start_quiz_form').on('submit',  function () {
+    $('.bottom-section').append('<div class="generating" tabindex="-1"><h4><%= I18n.t('idp.headings.generating') %></h4>' +
+      '<p><%= I18n.t('idp.body.generating') %></p></div>');
+    $('#start_quiz').addClass('spinner');
+    $('#start_quiz').attr('disabled', true);
+    $('.generating').focus();
+  });
+});
```

Comment:
right now I want to get a basic functioning app stood up asap so I don't mind if we do a few heavy loaded commits. Perhaps we should call the branch necessary-bits-for-functional-auth-flow (including id proofing)